src/codegeneration/CommonJsModuleTransformer.js

```diff
@@ -43,7 +43,7 @@ import {prependStatements} from './PrependStatements.js';
 
 export class CommonJsModuleTransformer extends ModuleTransformer {
 
-  constructor(identifierGenerator, reporter, options) {
+  constructor(identifierGenerator, reporter, options = undefined) {
```

Comment:
The AmdTransformer is created w/o options in one test: https://github.com/google/traceur-compiler/blob/master/test/unit/codegeneration/AmdTransformer.js#L30

I think the change is fine.